pkg/util/winutil/process.go

```diff
@@ -88,9 +90,12 @@ func ReadProcessMemory(h windows.Handle, from, to uintptr, count uint32) (bytesR
 		uintptr(unsafe.Pointer(&bytes)))
 
 	if r == 0 {
-		err = windows.GetLastError()
-		fmt.Printf("Error %v %v\n", err, e)
-		return
+		if e == windows.ERROR_ACCESS_DENIED {
+			log.Debugf("Access denied error getting process memory ")
```

Comment:
```suggestion
			log.Debugf("Access denied error getting process memory")
```

Also, this seems like an important error for a user so I would assume that it would probably be a `Warnf` and at a minimum an `Infof`

Reply:
it's an "expected" error.  There are a few processes (the system process, etc.), where we can't get rights.  Putting it in info or warn would spam the logs.

Reply:
Fair enough (the typo still need a fix though)